docs/utilities/batch.md

```diff
@@ -141,12 +141,23 @@ Processing batches from SQS works in three stages:
 
 #### FIFO queues
 
-When using [SQS FIFO queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html){target="_blank"}, we will stop processing messages after the first failure, and return all failed and unprocessed messages in `batchItemFailures`.
-This helps preserve the ordering of messages in your queue.
+When using [SQS FIFO queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-fifo-queues.html){target="_blank"}, a batch may include messages from different group IDs.
 
-```typescript hl_lines="1-4 8 20-22"
---8<-- "examples/snippets/batch/gettingStartedSQSFifo.ts"
-```
+By default, we will stop processing at the first failure and mark unprocessed messages as failed to preserve ordering. However, this behavior may not be optimal for customers who wish to proceed with processing messages from a different group ID.
+
+Enable the `skipGroupOnError` option for seamless processing of messages from various group IDs. This setup ensures that messages from a failed group ID are sent back to SQS, enabling uninterrupted processing of messages from the subsequent group ID.
+
+=== "Recommended"
+
+    ```typescript hl_lines="1-4 13 28-30"
+    --8<-- "examples/snippets/batch/gettingStartedSQSFifo.ts"
+    ```
+
+=== "Enabling skipGroupOnError flag"
+
+    ```typescript hl_lines="1-4 13 28-30"
+    --8<-- "examples/snippets/batch/gettingStartedSQSFifoSkipGroupOnError.ts"
+    ```
 
 1. **Step 1**. Creates a partial failure batch processor for SQS FIFO queues. See [partial failure mechanics for details](#partial-failure-mechanics)
```

Comment:
We need to indent this at the same level of the code block (one extra indent) otherwise it won't be rendered correctly.

I would also move this to ~L155 so it's only related to the first code block and remove the corresponding `// (1)!` annotation from the new `gettingStartedSQSFifoSkipGroupOnError.ts` code snippet

Reply:
```suggestion
    1. **Step 1**. Creates a partial failure batch processor for SQS FIFO queues. See [partial failure mechanics for details](#partial-failure-mechanics)
```

Reply:
One thing moving to `~L155` didn't render properly, i had to put it to `~L156` to make it render correctly.